server/sonar-web/src/main/js/apps/account/organizations/CreateOrganizationForm.js

```diff
@@ -229,7 +229,7 @@ class CreateOrganizationForm extends React.PureComponent {
           </div>
 
           <footer className="modal-foot">
-            {this.state.processing
+            {this.state.loading
```

Comment:
usually I disable the submit button and display the spinner, let's keep the consistency